### ISSUE TITLE
[Feature]: Support n (number of completions) in together_ai models

### ISSUE THREAD

**Issue by slobodan-nf** [BerriAI/litellm#6919]
### The Feature

I am able to set `n` (number of completions) to a number greater than 1 when using the together API directly, but not through litellm. I am using `Meta-Llama-3.1-8B-Instruct-Turbo`.

**Example using the together API:**
```py
from together import Together

client = Together(api_key="")

completion = client.chat.completions.create(
    model="meta-llama/Meta-Llama-3.1-8B-Instruct-Turbo",
    messages=[{"role": "user", "content": "What are the top 3 things to do in New York?"}],
    temperature=0.5,
    n=2,
)

print(len(completion.choices)) # prints 2
```

**Example using the litellm API:**
```py
import litellm
import os

os.environ["TOGETHERAI_API_KEY"] = ""

completion = litellm.completion(
    model="together_ai/meta-llama/Meta-Llama-3.1-8B-Instruct-Turbo",
    messages=[{"role": "user", "content": "What are the top 3 things to do in New York?"}],
    temperature=0.5,
    n=2,
)
print(len(completion.choices)) # raises an exception
```
The exception:
```
litellm.UnsupportedParamsError: together_ai does not support parameters: {'n': 2}, for model=meta-llama/Meta-Llama-3.1-8B-Instruct-Turbo. To drop these, set `litellm.drop_params=True` or for proxy:

`litellm_settings:
 drop_params: true`
```

### Motivation, pitch

I would like to use this feature as it saves costs, reduces latency and makes the implementation cleaner.

### Twitter / LinkedIn details

www.linkedin.com/in/slobodan-jenko-466370172

**Comment by ghost** [comment]
@slobodan-nf is this still happening? we recently fixed together ai optional param mapping

**Comment by ghost** [comment]
fixed since - v1.52.9 - https://github.com/BerriAI/litellm/commit/3beecfb0d42dd074078b5c87770c5fb0e5dd18a0#diff-5340907f012986dbaf4e5f8745d017520b80535825a2c3a025a40659ce9880bdR164
